src/System.Diagnostics.Process/ref/System.Diagnostics.Process.Manual.cs

```diff
@@ -10,8 +10,18 @@ namespace System.Diagnostics
     public partial class Process : System.IDisposable
     {
         public void Dispose() { }
+        /// <summary>
+        /// Release all resources used by this process.
+        /// </summary>
+        /// <param name="disposing">
+        /// true to release both managed and unmanaged resources; false to release only unmanaged resources.
```

Comment:
Interesting. Our IntelliSense files never used that tag before but our source content does have a tag called languagekeyword in these cases, so the langword could be added to the IntelliSense. I can open a bug to the VSC team but I don't think it would get fixed in time for our importing process. Also, does langword validates the content inside the tag? I'm asking because langword could have been applied to things that are not actually language keywords. But we do use that for true, false, null, etc.